configure.ac

```diff
@@ -74,6 +74,12 @@ fi
 AM_CONDITIONAL(ENABLE_TESTS_GLES,
     [test "x$enable_tests_gles" = "xyes"])
 
+HAVE_GTEST="no"
+ifdef([GTEST_LIB_CHECK],
```

Comment:
> make sure that it is only available when debug is enabled. If debug is not enabled we should not bother.

Actually, I disagree with this... we will want to be able to compile and execute the unit tests with both debug and non-debug since it is possible for developers to accidentally place code inside the __ENABLE_DEBUG__ guards that otherwise shouldn't be.